### ISSUE TITLE
add device: Tuya SOS alarm button

### ISSUE THREAD

**Issue by ithz4** [ioBroker/ioBroker.zigbee#730]
I bought [this](https://de.aliexpress.com/item/4001210398607.html) SOS Alarm-Button from Tuya.
The device was successfully added in iobroker but this model TS0215A differs from the supported button TS0218.
Both devices appear to be the same.
It is possible to adapt the configuration from TS0218 to the TS0215A since the device is not shown in the objects of the zigbee-instance?

Kind regards!

Details:
```
model: TS0215A
type: EndDevice
nwk: 52965
manuf id: 4098
manufacturer :_TZ3000_4fsgukof
power: Battery
app version: 67
hard version: 1
zcl version: 3
stack version: 0
date code:
interviewed: true
configured: false
endpoint: 1
profile: 260
input clusters:genBasic (0) genPowerCfg (1) ssIasZone (1280) ssIasAce (1281)
output clusters:genOta (25) genTime (10)
```

**Comment by kirovilya** [comment]
find zigbee-herdsman-converters in your installation. then find devices.js inside.
in this file find line with `zigbeeModel: ['TS0218']` and add there your model `zigbeeModel: ['TS0218', 'TS0215A']`
https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices.js#L10724
then restart adapter.

**Comment by ithz4** [comment]
Thank you very much for your advice!
I added the model `TS015A` in devices.js of my iobroker installation and restarted the adapter as well as the raspberry. 
When I push the button the following error message appears in the log:
```
Device ec1bbdfffeeXXXXX "TS0215A" not described in statesMapping.
```
Is there anything else I have to adjust?

**Comment by kirovilya** [comment]
Strange, it should have worked ...
Are you using the latest adapter? 1.2.1?

**Comment by ithz4** [comment]
I removed the zigbee instance and made a fresh installation from github.
Now everything works fine, thank you for your hints! :)
Do I have to modify the `devices.js` each time an update is served?

**Comment by kirovilya** [comment]
You need make PR to zigbee-herdsman-converters, like other users did that :)

**Comment by ithz4** [comment]
I made a PR 👍 
Thanks a lot for your help! :)

**Comment by flipjiz** [comment]
Hi guys.

I received my TS0215A today but I'm kinda lost.
I'm running newest iobroker version and newest (1.2.1)  Zigbee Adapter.
When I start pairing the timer just running down without any messages from the button between it.
I've added TS0215A to the devies.js and restarted like @kirovilya said. Changed loglevel to "silly" but nothing.
What you mean PR? Maybe some of you guys can help :(
Thank you very much!
